.github/workflows/pytorch181_cpu.yml

```diff
@@ -12,15 +12,17 @@ jobs:
   build:
     uses: ./.github/workflows/resuable_cpu_build.yml
     with:
-      remote_runtime_docker: bladedisc:latest-runtime-torch1.8.1-cpu
+      remote_runtime_docker: bladedisc:latest-runtime-torch-pre-cpu
       develop_base_image: nvidia/cuda:11.1.1-cudnn8-devel-ubuntu18.04
       runtime_base_image: nvidia/cuda:11.1.1-cudnn8-runtime-ubuntu18.04
+      extra_build_args: --build-arg PYTHON_VERSION=PYTHON3.8
       extra_envs: -e TORCH_BLADE_BUILD_WITH_CUDA_SUPPORT=OFF
-        -e TORCH_BLADE_CI_BUILD_TORCH_VERSION=1.8.1+cpu
+        -e TORCH_BLADE_CI_BUILD_TORCH_VERSION=pre+cpu
       exec_command: bash ./scripts/ci/test_pytorch_blade.sh
-      deploy_command: TORCH_VERSION=1.8.1+cpu bash ./scripts/ci/deploy_pytorch_blade.sh
+      deploy_command: TORCH_VERSION=pre+cpu bash ./scripts/ci/deploy_pytorch_blade.sh
+      host_tf_version: tensorflow==2.8.0
```

Comment:
need tf?

Reply:
the `reusable.yml` will try to install tensorflow,
<img width="1126" alt="image" src="https://user-images.githubusercontent.com/74886391/204206783-2071e74e-c818-4451-b501-d4d5c96bb129.png">
and I encountered the following bug if I don't specify the version
<img width="1454" alt="image" src="https://user-images.githubusercontent.com/74886391/204206976-b6f34520-e89f-42c1-8178-3e05ea681dc6.png">

Reply:
> need tf?

Why should we need a tf package in torch docker image?

Reply:
removed tf